helix-term/src/commands/typed.rs

```diff
@@ -1822,6 +1822,60 @@ fn run_shell_command(
     Ok(())
 }
 
+fn reset_change(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+    ensure!(args.is_empty(), ":reset-change takes no arguments");
+
+    let editor = &mut cx.editor;
+    let scrolloff = editor.config().scrolloff;
+
+    let (view, doc) = current!(editor);
+    if let Some(handle) = doc.diff_handle() {
```

Comment:
This isn't a super big block but the indentation here could be reduced:

```rs
let handle = match doc.diff_handle() {
    Some(handle) => handle,
    None => bail!("Diff is not available in the current buffer"),
};

// ...
```

and the same for the next `if let Some`

Reply:
that does look cleaner, thanks! This is actually an ideal case for `let ... else` but sadly we haven't reached a MSRV of 1.65 yet. Added a `TODO` for that